src/helpers/TypeHelpers.ts

```diff
@@ -1,29 +1,6 @@
 import { CommonModel } from '../models';
```

Comment:
All the unused import statements can be removed 😊

Reply:
Yeah, you have even lint error with that in testing workflow.